QuizApp/Client/Account/Models/UsernameUpdateRequestNetworkModel.swift

```diff
@@ -0,0 +1,12 @@
+struct UsernameUpdateRequestNetworkModel: Codable {
+
+    let name: String
+
+}
+
+extension UsernameUpdateRequestNetworkModel {
+
+    init(from repoModel: UsernameUpdateRequestRepoModel) {
+        self.init(name: repoModel.name)
+    }
```

Comment:
A layer may only know about layers below, not above.
Client can't know about anyone, and domain model can know about repo/DS model.

Mapping should go like this:
```swift
struct DataModel {

    let id: String

    func toModel() -> ClientModel {
        ClientModel(id: id)
    }

}

extension DataModel {

    init(from model: ClientModel) {
        self.init(id: model.id)
    }

}

```